docs/root/start/sandboxes/cache.rst

```diff
@@ -8,9 +8,9 @@ In this example, we demonstrate how HTTP caching can be utilized in Envoy by usi
 The setup of this sandbox is based on the setup of the :ref:`Front Proxy sandbox <install_sandboxes_front_proxy>`.
 
 All incoming requests are routed via the front Envoy, which acts as a reverse proxy sitting on
-the edge of the ``envoymesh`` network. Ports ``8000`` and ``8001`` are exposed by docker
+the edge of the ``envoymesh`` network. Port ``8000`` is exposed by docker
 compose (see :repo:`/examples/cache/docker-compose.yaml`) to handle ``HTTP`` calls
-to the services, and requests to ``/admin`` respectively. Two backend services are deployed behind the front Envoy, each with a sidecar Envoy.
```

Comment:
@yosrym93 could you confirm that there are no cache-specific stats to show the user ?

if there are ill add back and add docs/test

Reply:
No. The admin interface and its docs were copied as-is from the front proxy example. Feel free to remove this.

Reply:
if there are some cache-specific stats, i think we should show here

i couldnt see any - but i wasnt certain

Reply:
Do `setResponseFlag` and `setResponseCodeDetails` @ https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/http/cache/cache_filter.cc#L454 set any admin stats?

Reply:
not certain - but i guess i could be able to grep for some non-zero returned from cache code

ill have a look further

Reply:
I don't know anything about the admin interface. CacheFilter doesn't yet report stats (https://github.com/envoyproxy/envoy/issues/13413).

Reply:
ah great, thanks

ill remove for now, and we can revisit once the filter is reporting stats